src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private File file;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeDataToFile(readAndSort(fromFileName), toFileName);
+
+    }
+
+    public int [] readAndSort(String fromFileName) {
```

Comment:
The method name `readAndSort` does not accurately reflect what the method does. The method reads the data and calculates the statistics but does not sort anything. Consider renaming the method to better describe its purpose, such as `calculateStatistics`.